operations/enterprises.md

```diff
@@ -694,6 +694,97 @@ Adds a new task to the enterprise, optionally to a specified department.
 | --- | --- | --- | --- |
 | `TaskId` | string | required | Unique identifier of added task. |
 
+## Get all tasks
+
+Returns all tasks of the enterprise, filtered by identifiers or other filters.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/tasks/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "TaskIds": [
+        "65cf1aac-bef2-4653-9350-ab2600af65af"
+    ],
+    "DepartmentIds": [
+        "c28cfb42-a963-4195-ad26-ab1b009b6425"
+    ],
+    "ServiceOrderIds": [
+        "8D70F718-E19C-458D-8DDB-AB1B009B5487"
+    ],
+    "CreatedUtc": {
+        "StartUtc": "2019-12-08T00:00:00Z",
+        "EndUtc": "2019-12-10T00:00:00Z"
+    },
+    "ClosedUtc": {
+        "StartUtc": "2019-12-08T00:00:00Z",
+        "EndUtc": "2019-12-10T00:00:00Z"
+    },
+    "DeadlineUtc": {
+        "StartUtc": "2020-01-01T00:00:00Z",
+        "EndUtc": "2020-01-02T00:00:00Z"
+    },
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `TaskIds` | array of string | optional | Unique identifiers of tasks. |
+| `DepartmentIds` | array of string | optional | Unique identifiers of [Department](enterprises.md#department)s. Another filter must be submitted together with this one.  |
+| `ServiceOrderIds` | array of string  | optional | Unique identifiers of Service orders (for example a [Reservation](reservations.md#reservation) or [Product order](services#add-order)). |
+| `CreatedUtc` | [Time interval](enterprises.md#time-interval) | optional | Interval of task creation time. |
```

Comment:
ts date and time, not just time. The same bellow for all three intervals.
Does it make sense to use the sentence with the `within`?